### PR TITLE
Improved coqbot documentation in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by MSoegtropIMC**
This trivial PR improves the documentation of coqbot in CONTRIBUTING.md.

@Zimmi48 : please edit as you see fit.

### CODE REVIEW

**Review by Zimmi48 — APPROVED** [review]
Looks perfect. Thanks @MSoegtropIMC!

@coqbot merge now

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -792,11 +792,25 @@ organization, because of a limitation of GitHub).
...
-    for when you do not have access to a console).
+
+  - post a comment containing "@coqbot: merge now";
+    This is the recommended method and more convenient than the previous
+    script based method (see next bullet) e.g. for developers who do not have
+    a GPG key and for when you do not have access to a console.
+    "coqbot" will **not** check CI status - it is expected that the merger does
+    this manually upfront, but coqbot will deny the merge with an error
+    response in the following cases:
+    - no assignee
```
Comment (SkySkimmer) [inline CONTRIBUTING.md:803]: It's more specific than that: the author of the `merge now` comment must be an assignee

Comment (MSoegtropIMC) [reply CONTRIBUTING.md:803]: @Zimmi48 was quick with merging ... shall I do another PR?

Comment (Zimmi48) [reply CONTRIBUTING.md:803]: Right, sorry for merging too fast. Sure, another PR would be welcome.

**Pull request merged**